### PR TITLE
As/about

### PR DESCRIPTION
**Pull request by AndrewSimons3**
used flex box to design this page and it helped a ton!

### CODE REVIEW

File: src/About/about.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  background-color: #e4dfdf;
+  box-shadow: 0px 2px 10px 0px black;
+  padding: 10px 20px;
+  overflow: hidden;
+}
+
+.header-fix {
+  height: 65px;
+}
+
```
Comment (juan-rod) [inline src/About/about.css:38]: your content divs are a bit stiff. they don't move with the page size change. you can change that by adding flex to their parent element `.main-container`.  try:
`.main-container {`
`display: flex;`
    `flex-direction: column;`
    `justify-content: center;`
  `  align-items: center`
`}`

File: src/About/about.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  box-shadow: 0px 2px 10px 0px black;
+  opacity: .55;
+  color: black;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  width: 500px;
```
Comment (juan-rod) [inline src/About/about.css:53]: change the `width` to `max-width: 500px;`. at it's full width it'll 500px like you want. but in mobile it'll be whatever size it needs to be to fit the screen size.

**Review by juan-rod — APPROVED** [review]
great work! review the changes.  merge when ready

File: src/About/about.css (hunk 2 above)
Comment (AndrewSimons3) [reply src/About/about.css:53]: ah that makes sense!

**Pull request merged**